.E
======================================================================
ERROR: test_eval_2 (__main__.TestNetflix)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "TestNetflix.py", line 34, in test_eval_2
    print (sqrt(mean(square(subtract(predictions, actual)))))
NameError: name 'predictions' is not defined

----------------------------------------------------------------------
Ran 2 tests in 0.001s

FAILED (errors=1)
